general.py

```diff
@@ -0,0 +1,54 @@
+def addInts(intArray):
+    total = 0
+
+    for integer in intArray:
+        total += integer
+
+    return total
+
+def arrayIntersection(arrayA, arrayB):
+    returnArray = []
+
+    for i in arrayA:
+        if(i in arrayB):
+            returnArray.append(i)
+            arrayB.remove(i)
+            continue
+    
+    return returnArray
+
+def secondSmallest(intArray):

```

Comment:
nice use of recursion and solution 
However its quite difficult to read 
I did this:

```
    arr = set(arr)
    arr.remove(min(arr)) 
    return min(arr)
```

Reply:
see if you can speed this up by sorting the array by size